test/tasks/try-each-test.js

```diff
@@ -632,7 +632,7 @@ describe('tryEach', () => {
           ui: { writeLine: outputFn },
           project: { root: tmpdir },
           config,
-          commandArgs: ['ember', 'help', '--json', 'true'],
```

Comment:
`ember help --json` outputs a very long JSON string, which makes the output harder to read when testing locally.